### PR TITLE
Brady/add data to fhirstore

### PR DESCRIPTION
**Pull request by BradySkylight**
Tests are still a WIP as I figure out the complex mocking that needs to occur for my code in Python

### CODE REVIEW

**Comment by DanPaseltiner** [comment]
TODO: Move PHDI dependencies to that repository.

**Comment by celestemespinoza** [comment]
Task linked: [CU-2q4uy6b Add data to a GCP FHIR Store](https://app.clickup.com/t/2q4uy6b)

**Comment by DanPaseltiner** [comment]
This [PR](https://github.com/CDCgov/phdi/pull/19) in the `phdi` repo adds the GcpCredentialManager there. When that PR is merged we can update accordingly here.

**Comment by celestemespinoza** [comment]
Task linked: [CU-2q4uzhk Write GCP FHIR conversion function](https://app.clickup.com/t/2q4uzhk)

**Comment by celestemespinoza** [comment]
Task linked: [CU-2q4v07e Write GCP FHIR upload function](https://app.clickup.com/t/2q4v07e)

**Review by nickclyde — APPROVED** [review]
Looks great!

File: terraform/modules/cloud-functions/main.tf [hunk 1]
```diff
@@ -9,3 +9,15 @@ resource "google_cloudfunctions_function" "upcase" {
   trigger_http          = true
   entry_point           = "upcase_http"
 }
+
+resource "google_cloudfunctions_function" "upload-fhir-bundle" {
+  name        = "upload-fhir-bundle"
+  description = "Uploade a FHIR bundle to FHIR Store"
```
[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply terraform/modules/cloud-functions/main.tf:15]: fixed

File: cloud-functions/upload_fhir_bundle/main.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import json
+import functions_framework
+import flask
+from pydantic import BaseModel, ValidationError, validator
+from phdi.fhir.transport.http import upload_bundle_to_fhir_server
+from phdi.cloud.gcp import GcpCredentialManager
+
+
+class RequestBody(BaseModel):
+    """A class to modek the body of requests to the upload_fhir_bundle() function. The
```
[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply cloud-functions/upload_fhir_bundle/main.py:11]: fixed

**Pull request merged**